### DISCUSSION TITLE
同学们的 mini-vue

### DISCUSSION THREAD

**Discussion by cuixiaorui** [cuixiaorui/mini-vue#55] [General]
收集同学们自己完成的 mini-vue 库

**Comment by qinran0423** [comment]
https://github.com/qinran0423/mick-vue

**Reply by jp-liu** [reply]
占个楼: https://github.com/jp-liu/jp-mini-vue
哈哈哈

**Reply by bug-hou** [reply]
借一下楼主https://github.com/bug-hou/simple-vue

**Reply by Sean529** [reply]
借楼感谢  https://github.com/sean529/vscode-console-helper

**Reply by Hongbusi** [reply]
占个楼：https://github.com/Hongbusi/petite-vue

**Reply by xiaolu-coding** [reply]
借楼感谢 https://github.com/xiaolu-coding/mini-vue

**Reply by vancats** [reply]
https://github.com/lqf66/mini-vue 借楼

**Reply by TTiip** [reply]
https://github.com/TTiip/my-mini-vue 借楼

**Reply by Agony95z** [reply]
https://github.com/Agony95z/Vue3MiniSource 借楼

**Reply by Rafael-wxd** [reply]
https://github.com/Rafael-wxd/miniVue3.git 借楼

**Reply by liangjiasheng** [reply]
https://github.com/liangjiasheng/mini-vue 借楼

**Reply by scout-hub** [reply]
https://github.com/scout-hub/simplify-vue 借楼

**Reply by yekezz** [reply]
https://github.com/Enochzzz/mini_vue 借楼

**Reply by quyapeng** [reply]
https://github.com/quyapeng/vue-mini-project 借楼

**Reply by exwer** [reply]
借楼：https://github.com/love-loli/view
完整的ts类型 + vitest测试   谢谢大家

**Reply by rachern** [reply]
借楼 https://github.com/rachern/mini-vue3

**Reply by hzjswlgbsj** [reply]
借楼 https://github.com/hzjswlgbsj/simplified-vue

**Reply by X-swordx** [reply]
https://github.com/X-swordx/my-vue 借个楼谢谢

**Reply by SixZeroTwo** [reply]
借个楼栓Q~
https://github.com/SixZeroTwo/my-mini-Vue
刚刚撸完mini-vue，求个小星星🤭

**Reply by xinxin971225** [reply]
https://github.com/xinxin971225/ZIYU-MINI-VUE 占道

**Reply by kongyich** [reply]
借楼😁
https://github.com/konvyi/k-vue

**Reply by Aybuai** [reply]
https://github.com/Aybuai/mini_vue   借楼，求个小星星，后续还会持续更新博客文章，还有expand

**Reply by hzjswlgbsj** [reply]
自己在源码里加了很多注释，以及书籍中的思想，一起交流啊：https://github.com/hzjswlgbsj/simplified-vue

**Reply by roger922222** [reply]
欢迎 star
https://github.com/roger922222/roger-mini-vue

**Reply by Sunny-117** [reply]
每一节的具体实现https://github.com/learn-sources-code/mini-vue3/tree/main/mini-vue
<img width="621" alt="image" src="https://user-images.githubusercontent.com/73089592/199012824-5e4e672e-7b9d-4d03-8008-5bf15060ec3f.png">

**Reply by Seanxyc** [reply]
借楼，根据《vuejs设计与实现》持续完善 https://github.com/Seanxyc/my-vue3-core

**Reply by NansenHo** [reply]
欢迎 ⭐
https://github.com/NansenHo/mini-vue

**Comment by alexzhang1030** [comment]
https://github.com/alexzhang-dev/mini-vue

**Reply by zhenyuWang** [reply]
一楼人太多了： https://github.com/zhenyuWang/mini-vue3 🙏

**Reply by Hongbusi** [reply]
二楼抢个首位：https://github.com/Hongbusi/petite-vue

**Reply by TTiip** [reply]
二楼再借个位置 https://github.com/TTiip/my-mini-vue ～

**Reply by Agony95z** [reply]
https://github.com/Agony95z/Vue3MiniSource 借楼

**Reply by jinxxxxxi** [reply]
https://github.com/Zakisee/soft-vue 🙏🙏 😘

**Reply by liangjiasheng** [reply]
https://github.com/liangjiasheng/mini-vue 借楼

**Reply by myltx** [reply]
来来来 过来借个楼 蹭蹭 https://github.com/InkDyedGreenClothes/my-mini-vue

**Reply by rachern** [reply]
借楼 https://github.com/rachern/mini-vue3

**Reply by Aybuai** [reply]
https://github.com/Aybuai/mini_vue 借楼，求个小星星，后续还会持续更新博客文章，还有expand

**Reply by roger922222** [reply]
欢迎 star
https://github.com/roger922222/roger-mini-vue

**Reply by ghost** [reply]
借楼：
https://github.com/D-L-mm/guide-mini-vue

**Comment by You-5C24** [comment]
https://github.com/You-5C24/5c24-mini-vue

**Reply by briver0825** [reply]
借个楼哈哈, https://github.com/xiaozhuclassmate/mini-vue3

**Reply by Lonely-shang** [reply]
借个楼https://github.com/Lonely-shang/simple-vue

**Reply by Psilocine** [reply]
借楼 https://github.com/Psilocine/tiny-vue3
note 文件夹下是我整理的思路和重点
欢迎大佬们双向 start

**Reply by lwt09** [reply]
https://github.com/lwt09/mini-vue 挤挤

**Reply by HongxuanG** [reply]
借楼：https://github.com/HongxuanG/toy-vue3 🙏🙏 😘

**Reply by liangjiasheng** [reply]
https://github.com/liangjiasheng/mini-vue 借楼

**Reply by rachern** [reply]
借楼 https://github.com/rachern/mini-vue3

**Reply by Aybuai** [reply]
https://github.com/Aybuai/mini_vue 借楼，求个小星星，后续还会持续更新博客文章，还有expand

**Reply by roger922222** [reply]
欢迎 star
https://github.com/roger922222/roger-mini-vue

**Comment by chenfan0** [comment]
https://github.com/chenfan0/mini-vue3

**Reply by sicily621** [reply]
借个楼https://github.com/sicily621/guide-mini-vue

**Reply by JTR354** [reply]
https://github.com/JTR354/mini-vue3   谢谢

**Reply by yangwen97** [reply]
https://github.com/yangwen97/haierk-mini-Vue   谢谢

**Reply by Aybuai** [reply]
https://github.com/Aybuai/mini_vue 借楼，求个小星星，后续还会持续更新博客文章，还有expand

**Reply by roger922222** [reply]
欢迎 star
https://github.com/roger922222/roger-mini-vue

**Comment by xiaoxiayan** [comment]
https://github.com/xiaoxiayan/mini-Vue

**Reply by Aybuai** [reply]
https://github.com/Aybuai/mini_vue 借楼，求个小星星，后续还会持续更新博客文章，还有expand

**Comment by xxmyyds** [comment]
https://github.com/xxmyyds/my-mini-vue

**Comment by amebyte** [comment]
https://github.com/amebyte/mini-vue3-plus

**Comment by rhythm022** [comment]
https://github.com/rhythm022/jiangjj-frontend-studybook/tree/main/mini-vue

**Comment by Leiloloaa** [comment]
https://github.com/Leiloloaa/mini-vue3

**Reply by Leiloloaa** [reply]
欢迎star！有赞必回！

**Comment by quyapeng** [comment]
https://github.com/quyapeng/vue-mini-project

**Comment by littlewhitehhh** [comment]
https://github.com/littlewhitehhh/learning-mini-vue

**Comment by wenzheng-yi** [comment]
https://github.com/wenzheng-yi/tiny-vue

**Comment by briver0825** [comment]
https://github.com/xiaozhuclassmate/mini-vue3

**Comment by hidaviddong** [comment]
https://github.com/DongHY1/mini-vue3

**Comment by nikolausliu** [comment]
[mini-vue源码](https://github.com/nikolausliu/mini-vue)

**Reply by PLAxiaoxin** [reply]
https://github.com/PLAxiaoxin/mini-vue3

**Comment by luhaifeng666** [comment]
https://github.com/luhaifeng666/vue3-study

**Comment by jp-liu** [comment]
https://github.com/jp-liu/mini-vue-impl

**Comment by qc-z** [comment]
https://github.com/qc-z/mini-vue3

**Comment by Seanxyc** [comment]
https://github.com/Seanxyc/my-vue3-core.git

**Comment by myltx** [comment]
https://github.com/InkDyedGreenClothes/my-mini-vue

**Comment by XUBXUE** [comment]
https://github.com/XUBXUE/mini-vue

**Comment by abu-ab** [comment]
https://github.com/mynamexiaohui/mini-vue

**Comment by deweyou** [comment]
https://github.com/ouduidui/mini/tree/master/packages/mini-vue3

**Comment by yangyuhuan** [comment]
https://github.com/yangyuhuan/my-mini-vue

**Comment by CH0918** [comment]
https://github.com/CH0918/my-vue3

**Comment by Merlin218** [comment]
https://github.com/Merlin218/my-mini-vue

**Comment by Stan9726** [comment]
⭐https://github.com/Stan9726/Mini-Vue3

**Reply by fengjinlong** [reply]
https://github.com/fengjinlong/mini-vue-again

**Comment by hayeslv** [comment]
https://github.com/lvhaizhou/mini-vue

**Comment by sudongyuer** [comment]
速冻鱼儿的🚀🚀🚀mini-vue啊 star star 谢谢xdm💋  https://github.com/sudongyuer/mini-vue3

**Comment by captaino8** [comment]
https://github.com/captaino8/study-mini-vue3

**Comment by orzjose** [comment]
https://github.com/orzjose/mini-vue3

**Comment by sundada88** [comment]
https://github.com/sundada88/sun-vue3

**Comment by kongcodes** [comment]
KONG的mini-vue3🚀🚀🚀[https://github.com/kongcodes/mini-vue3](https://github.com/kongcodes/mini-vue3)

**Comment by cxyxxx0924** [comment]
陈新游的mini-vue 库 谢谢xdm！
https://github.com/cxyxxx0924/mini-vue

**Comment by Hongbusi** [comment]
https://github.com/Hongbusi/petite-vue

**Comment by xianjianlf2** [comment]
https://github.com/xianjianlf2/mini-vue

**Comment by xiaolu-coding** [comment]
https://github.com/xiaolu-coding/mini-vue

**Comment by JTR354** [comment]
follow > 收集同学们自己完成的 mini-vue 库

https://github.com/jtr354/mini-vue3

**Comment by zhenyuWang** [comment]
人好多啊，来了来了 😁
https://github.com/zhenyuWang/mini-vue3

**Comment by sicily621** [comment]
https://github.com/sicily621/guide-mini-vue

**Comment by jsy-0526** [comment]
https://github.com/jiangsongyang/mini-vue

**Comment by bug-hou** [comment]
https://github.com/bug-hou/simple-vue
一起混star

**Comment by AliceLi11** [comment]
https://github.com/AliceLi11/minivue3 star~star

**Comment by TTiip** [comment]
https://github.com/TTiip/my-mini-vue 兄弟们 冲鸭～

**Comment by masterX89** [comment]
https://github.com/masterX89/mini-vue 开卷～求个star

**Comment by sujianqingfeng** [comment]
https://github.com/sujianqingfeng/my-mini-vue 冲冲冲！

**Comment by Agony95z** [comment]
https://github.com/Agony95z/Vue3MiniSource 谢谢各位大佬，有赞必回

**Reply by bug-hou** [reply]
star了

**Comment by bug-hou** [comment]
https://github.com/bug-hou/simple-vue点赞必回

**Comment by czzonet** [comment]
https://github.com/czzonet/playground-mini-vue

**Comment by xsiqi-world** [comment]
https://github.com/xsiqi-world/mini-vue3  大佬们，有赞必会哈

**Reply by bug-hou** [reply]
star了

**Comment by yang49519845** [comment]
> https://github.com/yang49519845/yviewVue 
> start start 回start~

**Reply by bug-hou** [reply]
star了

**Comment by Btrya** [comment]
https://github.com/Btrya/pocket-vue
谢谢各位大佬！！有赞必回！！！

**Reply by bug-hou** [reply]
赞了

**Comment by Psilocine** [comment]
https://github.com/Psilocine/tiny-vue3

note 文件夹下是我整理的思路和重点

欢迎大佬们双向 start

**Reply by bug-hou** [reply]
赞了

**Comment by shuimu0579** [comment]
https://github.com/shuimu0579/vue3-mini-suimu
终于写完了mini-vue第一遍
欢迎各位互相star一波

**Reply by bug-hou** [reply]
赞了

**Reply by Befend** [reply]
https://github.com/Befend/tiny-series
记得回star哦

**Comment by whyjy** [comment]
https://github.com/whyjy
欢迎star！有赞必回！

**Reply by bug-hou** [reply]
赞了

**Reply by Befend** [reply]
https://github.com/Befend/tiny-series 
记得回star哦

**Reply by PLAxiaoxin** [reply]
https://github.com/PLAxiaoxin/mini-vue3 记得回我

**Comment by zhangchongyang01** [comment]
https://github.com/zhangchongyang01/minivue，请大家动动点赞的小手

**Comment by Rafael-wxd** [comment]
https://github.com/Rafael-wxd/miniVue3.git
完成了完成的vue3中的reactive模块

**Comment by Elben9** [comment]
https://github.com/Elben9/mini-vue🙏

**Comment by jinxxxxxi** [comment]
https://github.com/Zakisee/soft-vue 🙏🙏 😘

**Reply by Befend** [reply]
https://github.com/Befend/tiny-series 已star，记得回star哦

**Reply by bug-hou** [reply]
https://github.com/bug-hou/simple-vue 已赞

**Comment by sicily621** [comment]
https://github.com/sicily621/guide-mini-vue
欢迎star！有赞必回！

**Reply by bug-hou** [reply]
https://github.com/bug-hou/simple-vue 已赞

**Comment by liangjiasheng** [comment]
https://github.com/liangjiasheng/mini-vue
欢迎star！有赞必回！谢谢大佬们！

**Reply by bug-hou** [reply]
https://github.com/bug-hou/simple-vue 已赞

**Comment by Befend** [comment]
https://github.com/Befend/tiny-series 
欢迎star，有赞必回！

**Reply by bug-hou** [reply]
https://github.com/bug-hou/simple-vue 已赞

**Comment by ReadyPlayer67** [comment]
https://github.com/ReadyPlayer67/my-simplified-vue 求赞，有赞必回

**Reply by Befend** [reply]
https://github.com/Befend/tiny-series 已赞

**Reply by bug-hou** [reply]
https://github.com/bug-hou/simple-vue 已赞

**Comment by jobn123** [comment]
https://github.com/jobn123/mini-vue3 互赞

**Reply by bug-hou** [reply]
已赞

**Comment by yekezz** [comment]
https://github.com/Enochzzz/mini_vue

**Comment by laihaojie** [comment]
https://github.com/laihaojie/mini-vue  互赞

**Reply by bug-hou** [reply]
https://github.com/bug-hou/simple-vue 已赞

**Comment by exwer** [comment]
https://github.com/love-loli/view  
完整的ts类型注释 + vitest进行注释
谢谢！

**Comment by newplum** [comment]
https://github.com/newplum/mini-vue

**Comment by bug-hou** [comment]
https://github.com/bug-hou/simple-vue 互赞

**Comment by PLAxiaoxin** [comment]
https://github.com/PLAxiaoxin/mini-vue3

**Comment by pagnkelly** [comment]
[真不戳](https://github.com/pagnkelly/mini-vue3)

**Comment by rachern** [comment]
https://github.com/rachern/mini-vue3 求star

**Comment by hzjswlgbsj** [comment]
可以一起交流一下 https://github.com/hzjswlgbsj/simplified-vue

**Comment by CheneyDong123** [comment]
https://github.com/CheneyDong123/mini-vue3 有赞必回

**Comment by BigCoal** [comment]
https://github.com/BigCoal/min-vue3-code 找工作中，跪求点赞，希望可以找到好工作👍👍👍👍👍👍👍👍👍👍👍👍👍

**Reply by sujianqingfeng** [reply]
加油！

**Reply by BigCoal** [reply]
感谢

**Reply by sicily621** [reply]
https://github.com/sicily621/guide-mini-vue 已赞

**Comment by ReySnow** [comment]
https://github.com/ReySnow/mini-vue3

**Comment by lzy1960** [comment]
https://github.com/lzy1960/mini-vue3

**Comment by hzjswlgbsj** [comment]
自己在源码里加了很多注释，以及书籍中的思想，一起交流啊：https://github.com/hzjswlgbsj/simplified-vue

**Comment by X-swordx** [comment]
https://github.com/X-swordx/my-vue

**Reply by X-swordx** [reply]
最近准备跳槽，走过路过麻烦点个start，有赞必回

**Comment by yzhe819** [comment]
https://github.com/yzhe819/Be-Vue
项目会在完成基础的课程后完善课程没有涉及的TS类型
并且对于每一个完成的模块都撰写对应文档
如有错误，还请指正💖

**Comment by SixZeroTwo** [comment]
https://github.com/SixZeroTwo/my-mini-Vue
刚刚撸完mini-vue，求个小星星🤭

**Comment by xinxin971225** [comment]
https://github.com/xinxin971225/ZIYU-MINI-VUE
肝完拉，求个小星星

**Comment by HillBridge** [comment]
https://github.com/HillBridge/min-vue

**Comment by kongyich** [comment]
https://github.com/konvyi/k-vue
大佬们，求star😁

**Comment by Aybuai** [comment]
https://github.com/Aybuai/mini_vue   求个小星星，后续还会持续更新博客文章，还有expand

**Comment by lhp96** [comment]
https://github.com/lhp96/mini-vue 球球大家给偶点个Star 😎

**Comment by cuiyiming1998** [comment]
https://github.com/cuiyiming1998/vue3-mini
求个star

**Comment by roger922222** [comment]
https://github.com/roger922222/roger-mini-vue

**Reply by roger922222** [reply]
求star

**Comment by NinJa4270** [comment]
https://github.com/NinJa4270/mini-vue3
结课

**Comment by jjxs** [comment]
https://github.com/jjxs/core-vue
结课了，欢迎点赞

**Comment by viacooter** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by viacooter** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by 90ker** [comment]
https://github.com/90ker/my-mini-vue 结合《vue设计与实现》
从以下两个角度去完成源码，非常详细
- 模块
- 流程

**Comment by l1ax** [comment]
https://github.com/4noth1ng/my-mini-vue 持续更新中~ 包含个人的一些注释 <br>
每个模块会在掘金上做一点小总结
响应式部分： https://juejin.cn/post/7211713478178930725

**Comment by DanmoSAMA** [comment]
https://github.com/DanmoSAMA/my-mini-vue
求求大家的star啦

**Comment by rimutuyuan-feng** [comment]
https://github.com/cuixiaorui/mini-vue
求大家点点start，谢谢
